lldb/source/Interpreter/CommandInterpreter.cpp

```diff
@@ -3180,15 +3180,24 @@ void CommandInterpreter::IOHandlerInputComplete(IOHandler &io_handler,
   if ((result.Succeeded() &&
        io_handler.GetFlags().Test(eHandleCommandFlagPrintResult)) ||
       io_handler.GetFlags().Test(eHandleCommandFlagPrintErrors)) {
-    // Display any STDOUT/STDERR _prior_ to emitting the command result text
     GetProcessOutput();
 
+    // Display any inline diagnostics first.
+    if (!result.GetImmediateErrorStream() &&
+        GetDebugger().GetShowInlineDiagnostics() &&
+        line.find('\n') == std::string::npos) {
+      unsigned indent = GetDebugger().GetPrompt().size();
+      llvm::StringRef diags = result.GetInlineDiagnosticsData(indent, line);
+      PrintCommandOutput(io_handler, diags, true);
+    }
+
+    // Display any STDOUT/STDERR _prior_ to emitting the command result text.
```

Comment:
Did this comment move too far?  I'm pretty sure GetProcessOutput() is the call that displays STDOUT/STDERR.

Reply:
Maybe? I don't know the answer to this, but this isn't something that I changed in this patch.

Reply:
Yes, it definitely should bo before GetProcessOutput.  Your patch moved it from before that call to after both that call and your added text.

Reply:
Still need to fix this comment.